purchase_mass_mail/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+{
+    "name": "Purchase mass mail",
+    "summary": "Automatically send massive email to some purchase order",
+    "version": "14.0.1.0.0",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["yajo", "anddago78"],
+    "category": "Purchases",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "license": "AGPL-3",
```

Comment:
The rest of files have license LGPL (see line 2 of this same file).

So, is this LGPL or AGPL? cc @rafaelbn